src/lib/y2network/widgets/destination.rb

```diff
@@ -30,7 +31,7 @@ def opt
       def init
         Yast::UI.ChangeWidget(Id(widget_id), :ValidChars, Yast::IP.ValidChars + "-/")
         val = @route.to
-        self.value = val == :default ? "-" : val.to_s
+        self.value = val == :default ? "-" : ( val.to_s + "/" + val.prefix.to_s )
```

Comment:
If this operatio is common we should include it in helpers.

Reply:
yep, we will see how common it is. I think we should now design public API and then check how to DRY sources, where it makes sense.